helm/wasp-simple-h264-payload-parser/values.yaml

```diff
@@ -0,0 +1,23 @@
+config:
+  logLevel: info
+  # kafkaBrokers:
+  port: 80
+  waspSensorType: h264
+  kafkaOutTopic: video
+  kafkaPayloadRoutingPrefix: payloads
+  kafkaPayloadsReplicationFactor: 1
+  kafkaPayloadsPartitions: 5
+  init:
+    registerThingType: true
+    thingServiceName: wasp-thing-service
```

Comment:
Is this the correct name here?

Reply:
As a default this is fine.  The name can be over-ridden on an instance-by-instance basis.